### PR TITLE
Feature/events archive view

### PR DESCRIPTION
**Pull request by mike927**
Appended sorting feature for events archive view

### CODE REVIEW

**Comment by spidzio** [comment]
:-1:

**Comment by mike927** [comment]
Rails approach

File: BackEnd/app/controllers/events_controller.rb [hunk 1]
```diff
@@ -1,5 +1,17 @@
...
+  private
+
+  def sort_column
+    params[:sort] || 'time'
+  end
+
+  def sort_direction
+    params[:direction] || 'asc'
+  end
+end
```
Comment (spidzio) [inline BackEnd/app/controllers/events_controller.rb]: You should also store the sorting params in session. 
Right now, when I set sorting as name, then click page 2, I will get sorting by time again.

File: BackEnd/app/views/layouts/application.html.slim [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+    meta content=("text/html; charset=UTF-8") http-equiv="Content-Type" /
+    meta charset="utf-8" /
+    meta content="IE=edge" http-equiv="X-UA-Compatible" /
+    meta content="width=device-width, initial-scale=1" name="viewport" /
+    = stylesheet_link_tag 'application', media: 'all', 'data-turbolinks-track' => true
+    = javascript_include_tag 'application', 'data-turbolinks-track' => true
+    = csrf_meta_tags
+    title Events Archive
+  body
+    = yield
```
Comment (spidzio) [inline BackEnd/app/views/layouts/application.html.slim]: slim - yeah

File: BackEnd/app/controllers/events_controller.rb (hunk 1 above)
Comment (mike927) [inline BackEnd/app/controllers/events_controller.rb]: For me it works properly. You can inspect params in view to see that's OK

**Comment by spidzio** [comment]
:+1:

**Pull request merged**